test/unit/controller/edit-service-name/post.controller.test.js

```diff
@@ -56,11 +56,11 @@ describe('Controller: editServiceName, Method: get', () => {
         }
       }
       res = {}
-      const result = editServiceNameCtrl.post(req, res)
-      if (result) {
-        result.then(() => done()).catch(done)
-      } else {
-        done(new Error('Didn\'t return a promise'))
+
+      try {
+        await editServiceNameCtrl.post(req, res) 
+      } catch (err) {
+        throw new Error('Didn\'t return a promise')
```

Comment:
don't need the try/catch here I think. If there's an error thrown it will fail the test anyway.

Reply:
Removed the try/catch block.